libethereum/SkaleHost.cpp

```diff
@@ -376,9 +376,11 @@ ConsensusExtFace::transactions_vector SkaleHost::pendingTransactions(
 
             if ( tx.verifiedOn < m_lastBlockWithBornTransactions )
                 try {
```

Comment:
During normal operation, transactions with future nonces will not get here! So we don't need to set mtmEnabled here!
(or probably can even test nonce with assert)